compiler/src/iree/compiler/Codegen/Utils/Utils.cpp

```diff
@@ -1166,7 +1166,8 @@ computeDimUpperBound(Value shapedValue, unsigned dimNum,
             presburger::BoundType::UB, {shapedValue, dimNum},
             /*stopCondition=*/nullptr, /*closedUB=*/true);
     if (succeeded(maybeDimBoundSize))
-      return DimBoundSize{.baseSize = *maybeDimBoundSize, .scalable = false};
```

Comment:
I did have the suspicion when I saw the code. Because I've been wanting to write such code. The CI was happy, so I did not point it out.

Reply:
anyway, thanks for the quick fix!